tutorials/finetune_lora.md

```diff
@@ -167,17 +167,12 @@ python scripts/merge_lora.py \
   --out_dir "out/lora_merged/stablelm-base-alpha-3b/"
 ```
 
-> [!Note]
+> [!IMPORTANT]
 > If you changed the LoRA hyperparameters (`lora_r`, `lora_key`, etc.) in the
 > `finetune/lora.py` script, it is important to update the hyperparameter configuration
 > in the `scripts/merge_lora.py` script accordingly. Otherwise, you will encounter size
 > mismatch errors upon merging.
 
-> [!Note]
-> If the model was trained with quantized weights, it is possible to merge LoRA
-> and quantized pretrained weights by providing `--quantize` argument,
-> the same that was used for the finetuning stage.
-
```

Comment:
This is a leftover from another PR: the support for merging in `scripts/merge_lora.py` with quantization was dropped, but this part of the tutorial wasn't.
Not even mentioning that it was wrongly placed in the first place.